deployability/modules/allocation/README.MD

```diff
@@ -0,0 +1,444 @@
+## Allocation Module
+
+### User documentation
+
+The Allocation module allows you to create and destroy VMs both locally and in AWS, it also gives the possibility of creating macOS VMs on Wazuh infrastructure or PowerPC VMs, as long as you have the necessary permissions to do so. VMs can be AMD64, ARM64, Windows, macOS (Intel and ARM), and PowerPC (Centos and Debian).
```

Comment:
Does the current module deploy all those systems and architectures?

Also, Centos -> CentOS

Reply:
Done a81aec813